dvc/command/add.py

```diff
@@ -10,11 +10,11 @@
 class CmdAdd(CmdBase):
     def run(self):
         try:
-            if len(self.args.targets) > 1 and self.args.file:
+            if len(self.args.paths) > 1 and self.args.file:
```

Comment:
As we've discussed, this is very weird that we need to touch the args names here. Feels like we need an explicit way of telling how to complete this specific arg. E.g. maybe some special marker/flag to complete to files/dvcfiles/(in future we would even want stages, but there is no CLI command to provide that yet)

Reply:
sure, though on a related note for consistency's sake it's best to change these arg names.

Reply:
@casperdcl Sure, but let's do that in a separate PR, no need to mix it into this one.

Reply:
sure; reverted as no longer needed